recipes/ipyvuetify/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipyvuetify" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ece48f57f8ead8ca5dde2f53528d843f557ab2b5d18351636623e662b0d387e0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >= 7.0.0
```

Comment:
FYI: Spaces aren't allowed in version constraints so the correct way of writing this is:
```yaml
    - ipywidgets >=7.0.0
```
(I've fixed it in https://github.com/conda-forge/staged-recipes/commit/0c19fdc6dac8aeeb33b353e46b0a321758a281d4)